examples/browser/README.md

```diff
@@ -186,7 +186,7 @@ getrawmempool()
 
 Gets a list of unconfirmed transactions in memory.
 
-JSFiddle: [getrawmempool](https://jsfiddle.net/gh/get/jquery/3.0/cityofzion/neon-js/tree/master/examples/browser/getrawmempool/demo)
+JSFiddle: [getrawmempool](https://jsfiddle.net/gh/get/jquery/3.0/phetter/neon-js/tree/master/examples/browser/getrawmempool/demo)
```

Comment:
why do you want to revert it back to relying on `phetter` namespace as compared to `cityofzion` namespace?

Reply:
Thanks for catching this. I will resolve. That was from the old test code.